test-e2e/build/augment-path.test.js

```diff
@@ -0,0 +1,37 @@
+const childProcess = require('child_process');
+const path = require('path');
+const {promisify} = require('util');
+
+const {initFixture} = require('../test/helpers');
+
+const ESYCOMMAND = require.resolve('../../bin/esy');
+
+const promiseExec = promisify(childProcess.exec);
```

Comment:
not for this PR but soon we need to unify test utils between `test-e2e` and `test-e2e-esyi`